config/components/certmanager/ca-crt.yaml

```diff
@@ -0,0 +1,25 @@
+# We need create a self-signed certificate for the CA
```

Comment:
Can you explain more on this?

Are certificates not working for v0.11 without this?

Reply:
Is this needed for helm cert installation?

Reply:
>Is this needed for helm cert installation?

I haven't tried this yet.

Reply:
Added that piece for helm charts. I haven't tested that though.